common/api/core-common.api.md

```diff
@@ -330,6 +330,66 @@ export namespace AreaPattern {
     }
 }
 
+// @public
+export namespace Atmosphere {
+    export interface Props {
+        atmosphereHeightAboveEarth?: number;

```

Comment:
Anything that the user will not need to alter should be tagged internal
It sounds like `exposure` and `numInScatteringPoints` and `numOpticalDepthsPoints` should be the public controllable options.